owid/catalog/datasets.py

```diff
@@ -62,7 +65,7 @@ def create_empty(
     def add(
         self,
         table: tables.Table,
-        format: Literal["csv", "feather"] = "feather",
+        formats: List[str] = DEFAULT_FORMATS,
```

Comment:
This is perfectly fine, but can I ask why have you chosen simple list of options instead of Literal (or list of literals)? Are there any benefits / drawbacks to that?

Reply:
I was fighting the type checker, that's all. But I found an easy way around it, will put the type check back in now.